pkgs/servers/tt-rss/plugin-fever-api/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, fetchpatch, ... }: stdenv.mkDerivation rec {
+  pname = "tt-rss-plugin-fever-api";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "DigitalDJ";
+    repo = "tinytinyrss-fever-plugin";
+    rev = version;
+    sha256 = "000ys6r6a90x1v6fjyb86b0aab6j7g2d29w8d5sccyv895m4x8r1";
+  };
+
+  patches = [
```

Comment:
I kind of doubt this is actually needed. I'll deploy a version without it and test if it works.

Reply:
Yeah, this patch can be dropped. I probably added it during debugging, sorry.